tasks/run_agent.rb

```diff
@@ -0,0 +1,160 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
This seems to be the most sensible default.  Non AIO users can override this in their bolt configuration with [`interpreters`](https://puppet.com/docs/bolt/latest/bolt_configuration_options.html#global-configuration-options).